tests/test_mongodb_incremental_open_transactions.py

```diff
@@ -306,3 +330,8 @@ def test_run(self):
 
             # validate that the state value has not changed after deleting the bookmarked value in each collection
             self.assertEqual(state_2, state_3)
+
+            # Reset the transaction life time limit
+            LOGGER.info(f"Re-setting transaction life time limit to {original_life_time_limit} seconds")
```

Comment:
I've been informed that using f strings in LOGGER will fail pylint so I've been using:
LOGGER.info("Text %s, variable") format now

Reply:
I have not seen this issue. Also AFAIK pylist doesn't run on tests directory.

Reply:
Correct, it only runs on ui-automation / webapp side.  But I try to be consistent across both

Reply:
Removed the `f-string` and replaced it with `.format()`.